core/dbt/parser/hooks.py

```diff
@@ -76,6 +77,7 @@ def get_path(self) -> FilePath:
             project_root=self.project.project_root,
             searched_path='.',
             relative_path='dbt_project.yml',
+            modification_time=time.time(),  # this is wrong, but temporary
```

Comment:
Can we update the comment to be a little more descriptive? Why do we need this now, and under what conditions will it change?

Reply:
I've updated the comment.